travis/cfg/travis_run_pylint_pr.cfg

```diff
@@ -13,21 +13,29 @@ readme_template_url="https://github.com/OCA/maintainer-tools/blob/master/templat
 manifest_required_author="Odoo Community Association (OCA)"
 manifest_required_keys=license
 manifest_deprecated_keys=description,active
+attribute_deprecated=_fields,_defaults
```

Comment:
You are right.
Thank you
I excluded in `pylint_exclude_61.cfg` but we need to exclude in `pylint_exclude_70.cfg` too.